deepfence_server/pkg/integration/s3/s3.go

```diff
@@ -2,10 +2,10 @@ package s3
 
 import (
 	"bytes"
+	"compress/gzip"
```

Comment:
consider using https://github.com/klauspost/compress It is a drop in replacement and we have done benchmarks on it previously. It is faster and memory efficient when compared to standard library.